docs/ug/tutorial.md

```diff
@@ -1,14 +1,15 @@
 # Tutorial
 
-The following tutorial will guide you through the use of the Snitch cluster. You will learn how to develop, simulate, debug and benchmark software for the Snitch cluster architecture.
+The following tutorial will guide you through the use of the SNAX cluster. You will learn how to develop, simulate, debug and benchmark software for the SNAX cluster architecture.
```

Comment:
Again, there's no difference between snitch and SNAX.
To me SNAX is still an optional thing you would want to enable in the original project.
```suggestion
The following tutorial will guide you through the use of the Snitch cluster. You will learn how to develop, simulate, debug and benchmark software for the Snitch cluster architecture. Additionally, this guide explains how to attach and drive SNAX-based accelerators to the Snitch cluster.
```

Reply:
OK!